docs/notes/2.21.x.md

```diff
@@ -0,0 +1,152 @@
+# 2.21.x Release Series
+
+Pants 2 is a fast, scalable, user-friendly build system for codebases of all sizes. It's currently focused on Python, Go, Java, Scala, Kotlin, Shell, and Docker, with support for other languages and frameworks coming soon.
+
+Individuals and companies can now [sponsor Pants financially](https://www.pantsbuild.org/sponsorship).
+
+Pants is an open-source project that is not owned or controlled by any one company or organization, and does incur some expenses. These expenses are managed by Pants Build, a non-profit that was established for this purpose. This non-profit's only source of revenue is sponsorship by individuals and companies that use Pants.
+
+We offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/sponsorship), as well as individual sponsorships via [GitHub](https://github.com/sponsors/pantsbuild).
+
+## What's New
+
+### Highlights
+
+- Automatic retries of tests for all built-in backends.
+- Significant speed-ups when manipulating Python dependencies, especially huge ones like PyTorch.
+- When a `pants test ...` run in CI fails, the summary now includes suggested invocation to copy-paste and re-run locally.
```

Comment:
Please suggest more highlights!

Reply:
I'm personally very excited for the buildx/cacheing work and the __defaults__ fix, but I'm not sure they quiet qualify as global highlights.

Reply:
I think the `__defaults__` handling of target generators probably does, good call-out!